eZ/Publish/Core/Persistence/Legacy/Tests/_fixtures/schema.pgsql.sql

```diff
@@ -30,7 +30,8 @@ DROP TABLE IF EXISTS ezimagefile;
 CREATE TABLE ezimagefile (
     contentobject_attribute_id integer DEFAULT 0 NOT NULL,
     filepath text NOT NULL,
-    id SERIAL
+    id SERIAL,
+    version integer DEFAULT 0 NOT NULL
 );
```

Comment:
PostgreSQL fixture also contains indices, but defined at the end. Please add the missing index.